spec/Powertrain/CombustionEngine.vspec

```diff
@@ -18,6 +18,13 @@
 # Engine attributes
 #
 
+EngineCode:
+  datatype: string
+  type: attribute
+  description: Engine code designation, as specified by vehicle manufacturer.
+  comment: For hybrid vehicles the engine code may refer to the combination of
+           combustion and electric engine.
```

Comment:
Multiline.  Pending outcome of #405, but you could also just join the line into one IMHO.

Reply:
As I understand correct Yaml, see #405. 

Concerning if this shall be 1 or 2 lines it depends on what style guide we want to follow. Merging comment to 1 line gives a line that is about 110 characters long. With 2 lines we are within traditional 80 characters. Maybe we shall specify a recommended maximum line length in our documentation.